src/framework/theme/component/index.ts

```diff
@@ -1,3 +1,5 @@
 export * from './mapping';
 export * from './theme';
 export * from './style';
```

Comment:
If we're going to wrap everything in AppProvider, there is no need to export `StyleProvider` now

Reply:
This issue need discussing because there can be the case when user wants to wrap StyleProvider into custom "panels"